jerry-core/parser/js/js-parser-expr.c

```diff
@@ -339,6 +339,13 @@ parser_append_object_literal_item (parser_context_t *context_p, /**< context */
         break;
       }
 
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+      if (context_p->status_flags & PARSER_IS_CLASS)
+      {
+        break;
+      }
+#endif /* !CONFIG_DISABLE_ES2015_CLASS */
+
```

Comment:
Cannot we return early form this function? Or not even call it?

Reply:
True, calling it is unnecessary.